types/htmlhint/types.d.ts

```diff
@@ -0,0 +1,75 @@
+import HTMLParser from './htmlparser';
+import Reporter from './reporter';
+
+export as namespace htmlhint;
```

Comment:
this is wrong, exported global seems to be 'HTMLHint', also this is wrong place to do this (for UMD, index.d.ts). This is from umd package:
```js
global.HTMLHint
```